tests/shared.py

```diff
@@ -125,9 +125,9 @@ def get_driver_features(backend):
             features.add(protocol.Feature(raw))
         # TODO: remove this once all drivers manage the TLS feature flags
         #       themselves.
-        if get_driver_name() in ["java"]:
+        if get_driver_name() in ["java", "ruby"]:
             features.add(protocol.Feature.TLS_1_1)
-        if get_driver_name() in ["java", "dotnet"]:
+        if get_driver_name() in ["java", "dotnet", "ruby"]:
```

Comment:
Why do you want it to be added here as opposed to adding the feature flags in your back end? This code was only added to get the PR pipelines green and is supposed to be removed as soon (hopefully sooner than later) as the driver back ends have added it.